packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -213,6 +213,7 @@ export class HttpServer extends RunnableModule {
     });
     this.app.use(
       expressPromBundle({
+        buckets: [0.003, 0.03, 0.1, 0.3, 1.5, 3, 5, 8, 12, 20, 30],
```

Comment:
```suggestion
        buckets: [0.003, 0.03, 0.1, 0.3, 0.75, 1.5, 3, 5, 8, 12, 20, 30],
```
@blaggacao looking at this now, maybe we should this extra step

Reply:
rather safe than sorry! :-)

I'm sure there's a cost, but confident it's negligible.